lib/modules/dosomething/dosomething_campaign/dosomething_campaign.theme.inc

```diff
@@ -62,14 +62,18 @@ function dosomething_campaign_preprocess_node(&$vars) {
     return;
   }
 
-  // If the hot module is enabled and the high season is set on a campaign.
-  if (dosomething_campaign_is_hot_module($vars)) {
-    dosomething_campaign_preprocess_hot_module($vars, $wrapper);
-  }
 
-  // If the win module is enabled
-  if (dosomething_campaign_is_win_module($vars)) {
-    dosomething_campaign_preprocess_win_module($vars, $wrapper);
+  // If there are dates set, check hot/win states.
+  if (isset($vars['field_high_season'][0])) {
```

Comment:
Would it make sense to put this in `dosomething_campaign_is_hot_module` and `dosomething_campaign_is_win_module`, so those methods always return false if they don't have the proper variables to check?